src/mod/channels.mod/help/chaninfo.help

```diff
@@ -162,27 +162,21 @@
                        having ops.
         %bstatic%b         Allow only permanent owners to remove the channel?
      The following are flood settings, they are set via ".chanset <channel>
-     flood-type number:seconds", where number and second are integers
-     indicating the number of times in how many seconds the flood will be
-     triggered. Setting one of these to 0:0, 0:1, or 1:0 will deactivate the
-     respective flood setting.
+     <flood-type> <number>[:<seconds>]", where <number> and <seconds> are
```

Comment:
I'd leave the `[:<seconds>]` but would clarify further that only 0 can be used on it's own?